packages/core/helper-plugin/lib/src/components/ReactSelect/utils/getSelectStyles.js

```diff
@@ -1,4 +1,4 @@
-const getSelectStyles = theme => {
+const getSelectStyles = (theme, { 'aria-errormessage': error }) => {
```

Comment:
Mh, I'm not a fan of passing in the error message through aria. Should we pass it in via an `error` prop and do the `aria-` handling inside of `ReactSelect` instead?

Reply:
actually the `aria` works / is handled by `ReactSelect` I think (see below), so I don't think we need to add anymore more an `aria` side!

<img width="390" alt="image" src="https://user-images.githubusercontent.com/71838159/172572655-db90487d-86d2-413c-b6d7-937131435c9d.png">

for the error style, sure I can pass an error prop to getSelectStyles instead!

Reply:
What I meant was slightly different: I thought it might be helpful, if we pass an `error` prop to `ReactSelect` and render the `aria-errormessage` and `aria-invalid` props there, if an error was passed. This would make the component more compliant with the others and avoid some duplication in our forms ...

Reply:
sure will do!